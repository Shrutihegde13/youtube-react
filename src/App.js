import React, { Component } from 'react';
//import HeaderNav from './containers/HeaderNav/HeaderNav';
//import { SideBar } from './containers/Sidebar/SideBar';
import {Home} from './containers/Home/Home';
import {Watch} from './containers/Watch/Watch';
import {AppLayout} from './components/AppLayout/AppLayout';
//import './containers/Home/Home.scss';
import {Route,Switch} from 'react-router-dom';



class App extends Component {
  render() {
    return (
      <AppLayout>
        <Switch>
          <Route path="/watch" component={Watch}/>
          <Route path="/" component={Home}/>

      </Switch>
      </AppLayout>
         
    );
  }
}

export default App;
