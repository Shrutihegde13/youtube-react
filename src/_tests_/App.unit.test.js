import React from 'react';
import ReactDOM from 'react-dom';
import App from '../App';
import {shallow} from 'enzyme';

describe('App',()=> {
  test('renders',()=> {
    const wrapper = shallow(
      <App/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});