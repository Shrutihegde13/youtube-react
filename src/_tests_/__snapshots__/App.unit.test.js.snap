// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<AppLayout>
  <Switch>
    <Route
      component={[Function]}
      path="/watch"
    />
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</AppLayout>
`;
